### PR TITLE
feat: support dynamic values (JIT) & add tests

### LINKED ISSUE #53: Negative values not being generated
```
"tailwindcss": "^3.0.24",
"tailwindcss-rtl": "^0.9.0"
```

When using negative values (like `-ms-16`) the proper CSS is not being generated. I've just updated from Tailwind 2 and the plugin in version 0.7 where everything worked like charm.

### PR DESCRIPTION
**Pull request by vltansky**
closes #53 #51

### CODE REVIEW

File: src/insetUtilities.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
+module.exports = ({ matchUtilities, theme }) => {
+  matchUtilities(
+    {
+      start: (value) => ({
+        '[dir="rtl"] &': {
+          right: value,
+        },
+        '[dir="left"] &': {
+          left: value,
+        },
```
Comment (Sina7312) [inline src/insetUtilities.js:10]: Shouldn't this be `[dir="ltr"]`?

File: src/insetUtilities.js [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+      }),
+      end: (value) => ({
+        '[dir="rtl"] &': {
+          left: value,
+        },
+        '[dir="left"] &': {
+          right: value,
+        },
```
Comment (Sina7312) [inline src/insetUtilities.js:18]: This should be `[dir="ltr"]` as well.

File: tests/test.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+const postcss = require("postcss");
+const tailwindcss = require("tailwindcss");
+
+const rules = {
+  "ps-[JIT_VALUE]": `.ps-\\[JIT_VALUE\\] { padding-inline-start: JIT VALUE }`,
+  "pe-[JIT_VALUE]": `.pe-\\[JIT_VALUE\\] { padding-inline-end: JIT VALUE }`,
+  "ms-[JIT_VALUE]": `.ms-\\[JIT_VALUE\\] { margin-inline-start: JIT VALUE }`,
+  "me-[JIT_VALUE]": `.me-\\[JIT_VALUE\\] { margin-inline-end: JIT VALUE }`,
+  "start-[JIT_VALUE]": `[dir="rtl"] .start-\\[JIT_VALUE\\] { right: JIT VALUE }
+  [dir="left"] .start-\\[JIT_VALUE\\] { left: JIT VALUE }`,
```
Comment (Sina7312) [inline tests/test.js:10]: `[dir="ltr"]`

File: tests/test.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  "end-[JIT_VALUE]": `[dir="rtl"] .end-\\[JIT_VALUE\\] { left: JIT VALUE }
+[dir="left"] .end-\\[JIT_VALUE\\] { right: JIT VALUE }`,
```
Comment (Sina7312) [inline tests/test.js:12]: `[dir="ltr"]`

File: src/insetUtilities.js (hunk 2 above)
Comment (tonioriol) [reply src/insetUtilities.js:18]: Why this suggestion wasn't fixed? It needs to be `ltr` instead of `left`.

**Pull request merged**